forms/serializers/form.py

```diff
@@ -245,20 +239,77 @@ class Meta:
             ),
         ]
 
+    def entry_generator(self, entries, sections=None, fields=None):
+        if "sections" in entries:
+            yield from self.entry_generator(entries["sections"], sections=[entry for entry in entries["sections"]])
```

Comment:
Isn't these attributes now in the wrong order? Just by reading the code it seems like it.

In function definition there is entries before sections and if entries contains sections (the first if), there is the entries["sections"] feeded again as the first attribute. It smells like infinite loop to me. :thinking:

Reply:
so in my eyes it should yield like this:

`yield from self.entry_generator([entry for entry in entries["sections"], sections=entries["sections"]])`